### PR TITLE
[MOS-395] Add new def QCFG settings

### PR DESCRIPTION
**Pull request by mjan-dev**
**Description**

Add new default QCFG settings:
- servicedomain,
- nwscanmode.

This may help the issue, however we don't know for sure as we have no clear reproduction.
The solution was sourced from Quectel forums for similair issue. 

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [ ] Has documentation updated

### CODE REVIEW

File: module-cellular/at/src/Commands.cpp [hunk 1]
```diff
@@ -20,6 +20,8 @@ namespace at
             ret.push_back(AT::URC_UART1);
             ret.push_back(AT::AT_PIN_READY_LOGIC);
             ret.push_back(AT::CSQ_URC_ON);
+            ret.push_back(AT::SET_DEFAULT_SCANMODE);
+            ret.push_back(AT::SET_DEFAULT_SERVICEMODE);
```
Comment (SP2FET) [inline module-cellular/at/src/Commands.cpp:24]:
```suggestion
            ret.push_back(AT::SET_DEFAULT_SERVICEDOMAIN);
```

Comment (mjan-dev) [reply module-cellular/at/src/Commands.cpp]: My bad, changed by mistake before pushing.

**Review by pholat — APPROVED** [review]
Lets merge and check

**Review by SP2FET — APPROVED** [review]
The modem is properly registering to the network, no issues detected after quick testing

**Pull request merged**